portfolio/src/main/webapp/index.html

```diff
@@ -60,6 +60,17 @@ <h1>Jess's Portfolio</h1>
         <button onclick="addRandomGreeting()">Hello</button>
         <div id="greetingContainer"></div>
       </div>
+      <h2>Some Image Transformations!</h2>
+      <div class="speech-bubble card">
+        <button onclick="rotateProfilePicture()" class="buttonStyle">Rotate</button>
+        <button onclick="invertProfilePicture()" class="buttonStyle">Invert</button>
+        <button onclick="blurProfilePicture()" class="buttonStyle">Blur</button>
+        <button onclick="brightenProfilePicture()" class="buttonStyle">Brighten</button>
+        <button onclick="grayProfilePicture()" class="buttonStyle">Gray</button>
+        <button onclick="hueRotateProfilePicture()" class="buttonStyle">Hue-Rotate</button>
+        <button onclick="opacityProfilePicture()" class="buttonStyle">More Transparent</button>
+        <button onclick="resetProfilePicture()" class="buttonStyle">Reset</button>
```

Comment:
nit: if you want all buttons to be styled the same way, consider just applying the style globally instead of having each element use a `class`, i.e.

```css
button {
  /* ... */
}
```

(and as with previous changes, consider making this a list with `<ul>`)

Reply:
Thanks, all my buttons are the same so it makes sense to just use the button identifier.

I thought about using an unordered list, but decided against it because I thought it would be effect the layout of the buttons. I'll look into changing it.

Reply:
Yeah, the main reason to use `<ul>` is that it improves the semantic structure of the page, which should be conceptually distinct from its visual presentation (this is effectively a *list* of image transformations, similar to navigation being a *list* of links). Tools based on page structure alone, e.g. screenreaders, will usually work better when more semantically-appropriate elements are used.

More detail: https://developer.mozilla.org/en-US/docs/Glossary/Semantics#Semantics_in_HTML

I agree that having to override the default list style each time is a bit cumbersome :)